label_studio/data_export/api.py

```diff
@@ -59,7 +59,13 @@ def get_queryset(self):
         ],
         tags=['Export'],
         operation_summary='Export annotations',
-        operation_description='Export annotated tasks as a file in a specific format.',
+        operation_description='Export annotated tasks as a file in a specific format.'
+                              'For example, to export JSON annotations for a project to a file called annotations.json,'
+                              'run the following from the command line:'
+                              '```bash'
+                              'curl -X GET https://localhost:8080/api/projects/{id}/export?exportType=JSON '
+                              '-H \'Authorization: Token abc123\' --output annotations.json'
+                              '```',
```

Comment:
No idea if the syntax will show up properly when it's inside the description like this vs part of a comment but 🤞